plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -173,7 +168,7 @@ class Bazel(
             "mod", "graph", "--output", "json", "--disk_cache=", workingDir = projectDir
         )
 
-        val node = JSON.decodeFromString<BazelModule>(modGraphProcess.stdout)
+        val node = modGraphProcess.stdout.parseBazelModule()
```

Comment:
Don't model and lockfile model use different naming strategies? I believe to recall that the one uses snake-case and the other camel-case for keys. If that's the case, the serializer should not be shared.

Reply:
> Don't the model and lockfile use different naming strategies?

Also ping @haikoschol here to confirm (or not).

Reply:
The lock file uses camel-case and the output of `bazel mod graph` only contains module names and versions, no keys/attributes that would use either naming convention.

Reply:
I've double-checked the test fixtures, and indeed both file types seem to be using camel-case, see e.g. https://github.com/oss-review-toolkit/ort/blob/70f8de898e8801b853c9eaa8327e192925fb4851/plugins/package-managers/bazel/src/funTest/assets/fixtures/MODULE.bazel.lock#L2 and https://github.com/oss-review-toolkit/ort/blob/70f8de898e8801b853c9eaa8327e192925fb4851/plugins/package-managers/bazel/src/funTest/assets/fixtures/bazel_mod_graph_output.json#L10

Reply:
Ah right, I ran `bazel mod graph` to check, but forgot the JSON output flag. There are keys/attributes in that case.